doc/contribute/documentation/guidelines.rst

```diff
@@ -30,16 +30,40 @@ This document provides a quick reference for commonly used reST and
 Sphinx-defined directives and roles used to create the documentation
 you're reading.
 
+Content Structure
+*****************
+
+Tabs, spaces, and indenting
+===========================
+
+Indenting is significant in reST file content, and using spaces is
+preferred.  Extra indenting can (unintentionally) change the way content
+is rendered too.  For lists and directives, indent the content text to
+the first non-white space in the preceding line.  For example::
```

Comment:
eh :) This is probably worth fixing but I didn't want to clutter the diff even more for text that was already there.

Reply:
> I also personally don't mind if folks want to use double white space after the end of a sentence but it's true we don't have a guideline for this :)

I'd argue that it's semantically incorrect to do so. I believe it's from the earlier days (80s-90s) where it was commonly taught to people using PCs to better indicate start of sentences, but in the modern world how many pixels a space (` `) takes depends on the font and typesetting. Effectively it only ever made sense for monospace fonts and a long time ago. 

For example if I  started typing with  a mix of   how many spaces I  had between word  then that'd just  be weird, right? :)

Reply:
> (had to put that in quotes because Github actually treats multiple spaces in text as a single space).

So does Sphinx/ReST :) which is why I said I don't mind too much (as in: yes, it should probably be avoided but it doesn't really impact the rendered output).